### ISSUE TITLE
revise old test, 'Clicking on label search suggestion works'

### ISSUE THREAD

**Issue by cch5ng** [webcompat/webcompat.com#1138]
a previous search-non-auth.js test was removed because it was causing repeated local test failures.

> [the old test] it assumes that there will be a bug with an android label on the /issues page
> We should probably re-write it to just click on whatever label it finds, make a reference to its text, and then assert that we ended up with search results that match.

for ref, the old test contents:

```
'Clicking on label search suggestion works': function() {
              return this.remote
                .setFindTimeout(intern.config.wc.pageLoadTimeout)
                .get(require.toUrl(url('/issues')))
                .findByCssSelector('[data-remotename=browser-android]').click()
                .end()
                // click the first suggestion, which is "android"
                .findByCssSelector('.wc-IssueList:nth-child(1) > div:nth-child(2) > span:nth-child(1) > a:nth-child(1)').getVisibleText()
                .then(function(text) {
                  assert.include(text, 'android', 'Clicking on a suggested label gets you results.');
                })
                .end();
            },
```

(test would fail b/c sometimes an issue with android label could not be found)

for the test removal context, see #1135

**Comment by cch5ng** [comment]
I should be able to work on this

**Comment by miketaylr** [comment]
cf. #1536.

**Comment by miketaylr** [comment]
I believe @zoepage effectively fixed this in #2191.
